### ISSUE TITLE
Updater is suggesting the latest Spotify update is "Techbigs_Spotify".

### ISSUE THREAD

**Issue by theresyourproblem** [2Ra66it/updater-for-spotify#25]
I looked it up online and it looks like it's cracked Spotify Premium or some other sketchy modified version. Really confused. I already pay for premium, just need plain stock Spotify.

I'd also like to know if this is the direction that the updater is heading toward, which is fine, I'll just update through Aurora Store instead because I prefer to use untampered copies.

![image](https://user-images.githubusercontent.com/95443363/144490905-16b586bd-f0b9-4295-a06e-1488089c27a9.png)

**Comment by 2Ra66it** [comment]
Hey! This is a very strange situation, until the last moment there was the original Spotify, but apparently the Aptiode from which I receive updates began to distribute the modified version. If the situation does not change, I will have to migrate to the new API

**Comment by theresyourproblem** [comment]
Yeah, last thing you need is everyone using it getting weirded out by sketchy app downloads and no longer using it, and F-Droid possibly taking your app down for distributing cracked/modified copies of stuff.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
This is occurring currently.
